app/facades/quote_facade.rb

```diff
@@ -0,0 +1,8 @@
+class QuoteFacade
+
+ def self.quote_search(search_params)
```

Comment:
So this is the place that we would want to create our poro so that we can take the json data and make objects out of them. So if I were to refactor this:
```
search_data = QuoteService.search(search_params)
total_count = search_data[:totalCount]
quotes = search_data[:results].map do |quote_data|
   Quote.new(quote_data) <---- This is what creates our PORO
end
[total_count, quotes]
end